src/backend/distributed/planner/multi_explain.c

```diff
@@ -182,6 +182,8 @@ MultiExplainOneQuery(Query *query, IntoClause *into, ExplainState *es,
 		return;
 	}
 
+	ExplainOpenGroup("Distributed Query", NULL, true, es);
```

Comment:
Mostly that ExplainOnePlan used to generate the explain output for the merge query already generates { .. }, so wrapping it twice { { .. } } would not be valid. We could work around this by appending a field name without calling OpenGroup, but it becomes tricky to maintain for all the different formats. Technically, a query could also result in multiple plans, which is why the overall explain plan is also wrapped in an array, though that would not happen here since we explicitly generate a single plan.